.github/workflows/rustdocs.yml

```diff
@@ -0,0 +1,39 @@
+name: Rust docs
+on:
+  push:
+    tags:
+      - 'v*'
+
+
+jobs:
+
+  build-rust-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2021-04-22
+          target: wasm32-unknown-unknown
+          default: true
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - name: Install vercel
+        run: npm i -g vercel
+
+      - name: Build rust docs
+        run: cargo doc --all --no-deps --release
+
+      - name: Deploy rust docs
+        run: |
+          mv vercel.json target/doc
+          mv target/doc target/${GITHUB_REPOSITORY#*/}
+          cd target/${GITHUB_REPOSITORY#*/}
```

Comment:
As above mentioned, a CLI argument or flag to set the directory name instead of moving files around would be great (if `vercel-cli` provide this).

Otherwise, better add a line of comment if `vercel-cli` requires a specific directory.

Reply:
Yes, This step is required, I have already written a document in notion.

Reply:
So I prefer adding a comment here. But that's just my style. It depends on you.